### PR TITLE
[ADI] Send mail notification when a note is finished

### PR DESCRIPTION
**Pull request by ottaviano**

### CODE REVIEW

File: tests/Command/NotifyIdeaAuthorCommandTest.php [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+        $this->loadFixtures([
+            LoadIdeaData::class,
+        ]);
+
+        $this->runCommand('idea-workshop:notification:idea-author');
+
+        $this->assertCountMails(1, IdeaFinalizeMessage::class, 'jacques.picard@en-marche.fr');
+    }
+
+    public function testSendMailWhenNoteWillBeFinishedIn3Days()
```
Comment (Remg) [inline tests/Command/NotifyIdeaAuthorCommandTest.php]: `: void`

File: tests/Command/NotifyIdeaAuthorCommandTest.php [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+use Tests\AppBundle\Controller\ControllerTestTrait;
+
+/**
+ * @group command
+ */
+class NotifyIdeaAuthorCommandTest extends WebTestCase
+{
+    use ControllerTestTrait;
+
+    public function testSendMailWhenNoteIsFinished()
```
Comment (Remg) [inline tests/Command/NotifyIdeaAuthorCommandTest.php]: `: void`

File: tests/Command/NotifyIdeaAuthorCommandTest.php (hunk 1 above)
Comment (ottaviano) [reply tests/Command/NotifyIdeaAuthorCommandTest.php]: done

File: tests/Command/NotifyIdeaAuthorCommandTest.php (hunk 2 above)
Comment (ottaviano) [reply tests/Command/NotifyIdeaAuthorCommandTest.php]: done

**Pull request merged**